.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,39 @@
-## Description
-
-Describe your changes in detail.
-
-## Motivation and Context
-
-Why is this change required? What problem does it solve?
```

Comment:
I like the original Description and Motivation and Context chapters. Why changing them?